src/main/features/directions-questionnaire/routes/expert-evidence.ts

```diff
@@ -0,0 +1,45 @@
+/* tslint:disable:no-default-export */
+import * as express from 'express'
+import { Paths } from 'directions-questionnaire/paths'
+import { DirectionsQuestionnaireDraft } from 'directions-questionnaire/draft/directionsQuestionnaireDraft'
+import { Draft } from '@hmcts/draft-store-client'
+import { ExpertEvidence } from 'directions-questionnaire/forms/models/expertEvidence'
+import { Form } from 'forms/form'
+import { FormValidator } from 'forms/validation/formValidator'
+import { ErrorHandling } from 'shared/errorHandling'
+import { DraftService } from 'services/draftService'
+import { YesNoOption } from 'models/yesNoOption'
+
+function renderPage (res: express.Response, form: Form<ExpertEvidence>) {
+  res.render(Paths.expertPage.associatedView, { form: form })
+}
+
+export default express.Router()
+  .get(Paths.expertPage.uri, (req: express.Request, res: express.Response) => {
+    const draft: Draft<DirectionsQuestionnaireDraft> = res.locals.draft
+    renderPage(res, new Form<ExpertEvidence>(draft.document.expertEvidence))
+  })
+
+  .post(Paths.expertPage.uri,
+    FormValidator.requestHandler(ExpertEvidence, ExpertEvidence.fromObject),
+    ErrorHandling.apply(async (req: express.Request, res: express.Response) => {
+      const form: Form<ExpertEvidence> = req.body
+
+      if (form.hasErrors()) {
+        renderPage(res, form)
+      } else {
+        const draft: Draft<DirectionsQuestionnaireDraft> = res.locals.draft
+        const user: User = res.locals.user
+
+        draft.document.expertEvidence = form.model
+
+        await new DraftService().save(draft, user.bearerToken)
+
+        if (form.model.expertEvidence.option === YesNoOption.YES.option) {
+          res.redirect(Paths.expertYesPage.evaluateUri({ externalId: res.locals.claim.externalId }))
```

Comment:
These pages don't exist yet and may not be called this, I'd prefer just redirecting to the dashboard and letting who ever builds the next pages to do the linking.

Reply:
I thought i am being more helpful :), i have written redirect test, added paths for next page to Paths. So, whoever is picking up next just have to change name to whatever they like :)....